File: client2/mymuse/src/types/photo.ts
export interface CreatePhotoData {
    title:string;
    detailed_title: string;
    content:string;
    height: number;
    width: number;
    order: number;
    tag_id: number;
}

export interface UpdatePhotoData {
    id: number
    title:string;
    detailed_title: string;
}

File: server/internal/usecase/input/photo.go
package input

type CreatePhotoInput struct {
	Title         string
	DetailedTitle string `json:"detailed_title"`
	Content       string
	Height        int
	Width         int
	Order         int
	TagID         uint `json:"tag_id"`
}

type UpdatePhotoInput struct {
	ID            uint   `json:"id"`
	Title         string `json:"title"`
	DetailedTitle string `json:"detailed_title"`
}

type PhotoOrder struct {
	ID    uint `json:"id"`
	Order int  `json:"order"`
}

type PhotoOrderList []PhotoOrder
